### ISSUE TITLE
RasPi 2, MotionEyeOS 20171003, networking issues with eth0 and wlan0

### ISSUE THREAD

**Issue by quint21** [ccrisan/motioneyeos#1270]
I've tested the same camera (an Arducam UC-261 rev C) on a Pi 3, and two Pi ZeroW boards, using MotionEyeOS 20171003 (using the appropriate MotionEyeOS image for each board). I've also tested it on two Raspberry Pi 2s, using the same camera, and several wifi dongles (two Edimax EW-7811Un Realtek RTL8188CUS based dongles, and a generic RTL8188CUS dongle.) I've also installed MotionEye on a DietPi installation on the Raspberry Pi 2 boards (the latest version, based on Stretch).

The Pi ZeroW boards, and the Pi 3 all work perfectly with MotionEyeOS, connecting both over wifi, and ethernet (in the case of the Pi 3). The camera works perfectly using regular mode, and Fast Network Camera mode, providing smooth fluid motion and performance. 

The Pi 2 works fine running MotionEye on DietPi, using the normal mode of the camera. Network performance is good, both on the ethernet and wifi interface.

Running MotionEyeOS on the Pi 2 gives me very spotty networking performance. Fast network camera doesn't work. Normal camera mode seems to drop a lot of frames. When I ping the Pi 2 from my Windows computer, there are lots of dropped packets. I have a monitor connected to the Pi 2, and logged into it. I can't consistently ping my router from the Pi. (41 packets transmitted, 1 received, 97% packet loss.) I have tried having both ethernet and wifi connected at the same time, and only one interface- with the same results. The Pi pulls a DHCP address from the router (a TP-Link Archer C9), but beyond that the network is barely usable.

I re-downloaded, and re-flashed the 20171003 Pi 2 image. The SHA1 for the file is the same: d998b3507fc5844824726927351e152ebffb300d

To sum up: networking in the latest official release of MotionEyeOS is giving me trouble on a Pi 2, but not on a Pi ZeroW, or a Pi 3. The networking works ok when I use DietPi, a Raspbian-based OS. I've used the same camera on all systems.

[messages.log](https://github.com/ccrisan/motioneyeos/files/1549433/messages.log)
[dmesg.log](https://github.com/ccrisan/motioneyeos/files/1549436/dmesg.log)
[boot.log](https://github.com/ccrisan/motioneyeos/files/1549438/boot.log)
[motioneye.log](https://github.com/ccrisan/motioneyeos/files/1549441/motioneye.log)
[streameye.log](https://github.com/ccrisan/motioneyeos/files/1549444/streameye.log)
